src/init/replace_vulnerability_detector.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright (C) 2023, Wazuh Inc.
+# Shell script that updates 'vulnerability-detector' to 'vulnerability-detection'
+# Author: Kevin Cardenas <kevincardenasmiranda9@gmail.com>
+
+isVulnerabilityDetectionInstalled()
+{
+    local OSSEC_CONFIGURATION_FILE="$1"
+    local VULNERABILITY_DETECTION_PATTERN="<vulnerability-detection>"
+
+    if ( grep -q "$VULNERABILITY_DETECTION_PATTERN" "$OSSEC_CONFIGURATION_FILE" ); then
+        return 0
+    fi
+
+    return 1
+}
+
+updateVulnerabilityDetector()
+{
+    if [ "$INSTYPE" = "server" ]; then
+        local OSSEC_CONFIGURATION_FILE="$1"
+        if isVulnerabilityDetectionInstalled "$OSSEC_CONFIGURATION_FILE"; then 
+            return 0
+        fi
+        local OSSEC_CONFIGURATION_FILE_TMP="$1.tmp"
+
+        local VULN_TEMPLATE="./etc/templates/config/generic/wodle-vulnerability-detection.manager.template"
+        local INDEXER_TEMPLATE="./etc/templates/config/generic/wodle-indexer.manager.template"
+
+        touch $OSSEC_CONFIGURATION_FILE_TMP
+        local OSSEC_CONFIGURATION_FILE_BEFORE_VD="$(sed -ne '/<vulnerability-detector>/q;p' $OSSEC_CONFIGURATION_FILE)"
+        local OSSEC_CONFIGURATION_FILE_AFTER_VD="$(sed -e '1,/<\/vulnerability-detector>/d' $OSSEC_CONFIGURATION_FILE)"
+        # OSSEC_CONFIGURATION_FILE_BEFORE_VD
```

Comment:
Nitpick: add new line
```suggestion

        # OSSEC_CONFIGURATION_FILE_BEFORE_VD
```

Reply:
done